JuiceMaker/JuiceMaker/Controller/ViewController.swift

```diff
@@ -1,18 +1,80 @@
 //
 //  JuiceMaker - ViewController.swift
-//  Created by yagom. 
+//  Created by uemu, Toy.
 //  Copyright © yagom academy. All rights reserved.
 // 
 
 import UIKit
 
-class ViewController: UIViewController {
+final class ViewController: UIViewController {
+    
+    @IBOutlet weak var strawberryLabel: UILabel!
+    @IBOutlet weak var bananaLabel: UILabel!
+    @IBOutlet weak var pineappleLabel: UILabel!
+    @IBOutlet weak var mangoLabel: UILabel!
+    @IBOutlet weak var kiwiLabel: UILabel!
+    
+    var juiceMaker = JuiceMaker()
+    var fruitStore = FruitStore.shared
+    var fruitJuice: String?
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
+        showStock()
     }
 
-
+    func showStock() {
+        fruitStore.fruitQuantity.forEach{(key, value) in
+            switch key {
+            case .strawberry:
+                strawberryLabel.text = String(value)
+            case .banana:
+                bananaLabel.text = String(value)
+            case .kiwi:
+                kiwiLabel.text = String(value)
+            case .mango:
+                mangoLabel.text = String(value)
+            case .pineapple:
+                pineappleLabel.text = String(value)
+            }
+        }
+    }
+    
+    @IBAction func manufactureJuice(_ sender: UIButton) {
+        guard let order = sender.currentTitle else { return }
+        guard let juice = FruitJuice.Order(rawValue: order)?.juiceOrder else { return }
+        do {
+            try juiceMaker.manufactureFruitJuice(in: juice)
+            fruitJuice = juice.rawValue
+            showStock()
+            successmanufactureJuice()
+        } catch {
+            failmanufactureJuice()
+        }
+    }
+    
+    func failmanufactureJuice() {
+        let alert = UIAlertController(title: "재고가 모자라요.", message: "재고를 수정할까요?", preferredStyle: .alert)
+        let success = UIAlertAction(title: "예", style: .default) { _ in
+            guard let vc = self.storyboard?.instantiateViewController(identifier: "FruitQuantityViewController") else { return }
+            vc.modalPresentationStyle = .fullScreen
+            self.present(vc, animated: true)
+        }
+        let cancel = UIAlertAction(title: "아니오", style: .cancel) { _ in }
+        
+        alert.addAction(success)
+        alert.addAction(cancel)
+        
+        self.present(alert, animated: true, completion: nil)
+    }
+    
+    func successmanufactureJuice() {
+        let alert = UIAlertController(title: "\(fruitJuice ?? "") 나왔습니다!", message: "맛있게 드세요!", preferredStyle: .alert)
+        let success = UIAlertAction(title: "예", style: .default) { _ in }
```

Comment:
```suggestion
        let success = UIAlertAction(title: "예", style: .default) 
```

<img width="983" alt="스크린샷 2023-09-20 오후 3 41 49" src="https://github.com/yagom-academy/ios-juice-maker/assets/59365211/74728bb8-7908-4be5-8be6-dabd2076d9e0">

UIAlertAction 의 initializer 를 살펴보면 handler 는 옵셔널입니다. 핸들러를 따로 사용하지 않는 거라면 생략이 가능합니다.

Reply:
핸들러를 사용하지 않기 때문에 생략 하였습니다.